src/cms.h

```diff
@@ -13,19 +13,45 @@
 #define CMS_FREE(ptr) free(ptr)
 #endif
 
-typedef struct
-{
+typedef unsigned int uint;
```

Comment:
:(

Reply:
Maybe something like `typedef uint32_t CMSHashType` ?

Reply:
It isn't just for the hash but the array itself.
I changed it to uint32_t as requested